tests/Unit/Domain/Asset/ValueObject/UploadIdTest.php

```
@@ -27,6 +27,16 @@ public function itStoresValueWhenUuidV4IsValid(): void
         self::assertSame($value, $uploadId->value);
     }
 
+    #[Test]
+    public function itGeneratesAValidUuidV4WhenRequested(): void
+    {
+        // Act
+        $uploadId = UploadId::generate();
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Domain\Asset\ValueObject\UploadId' in method 'itGeneratesAValidUuidV4WhenRequested'. (undefined)

(StaticAccess)